vng_api_common/notifications/models.py

```diff
@@ -1,97 +0,0 @@
-import uuid
```

Comment:
Same comment as for notification labels.

Reply:
added it to the warning in changelog as well